cmd/platform/channel.go

```diff
@@ -205,7 +219,7 @@ func addUserToChannel(channel *model.Channel, user *model.User, userArg string)
 	}
 }
 
-func deleteChannelsCmdF(cmd *cobra.Command, args []string) error {
+func archiveChannelsCmdF(cmd *cobra.Command, args []string) error {
```

Comment:
Hi @cpanato I was just looking at this for the docs, and I noticed that in this function, "delete" is used in a couple of places:
``` golang
	if len(args) < 1 {
		return errors.New("Enter at least one channel to delete.")
	}

	channels := getChannelsFromChannelArgs(args)
	for i, channel := range channels {
		if channel == nil {
			CommandPrintErrorln("Unable to find channel '" + args[i] + "'")
			continue
		}
		if result := <-app.Srv.Store.Channel().Delete(channel.Id, model.GetMillis()); result.Err != nil {
			CommandPrintErrorln("Unable to delete channel '" + channel.Name + "' error: " + result.Err.Error())
		}
	}
```
Should that be changed to "archive"?
Like this:
`return errors.New("Enter at least one channel to archive.")`
and
`CommandPrintErrorln("Unable to archive '" + channel.Name + "' error: " + result.Err.Error())`

Also, I noticed that the archive command seems to be available in all editions, but the restore command is available in Enterprise only. Is that intentional?